index.bs

```diff
@@ -119,8 +162,18 @@ Sensor use cases</a></cite> documents.
 Security and Privacy Considerations {#security-and-privacy}
 ===================================
 
-There are no specific security and privacy considerations
-beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
+Sensor readings provided by inertial sensors, such as accelerometer, could be used by adversaries
+to exploit various security threats, for example, [=keylogging=], [=location tracking=],
+[=fingerprinting=] and [=user identifying=].
+
+Research papers published by security community, for instance, [[KEYSTROKEDEFENSE]], indicate that
+by throttling the frequency, risks of successful attacks is not fully eliminated, while may greatly
+affect usefulness of a web application with ligitimate reasons to use the sensors.
+
+The [[TOUCHSIGNATURES]] and [[ACCESSORY]] research papers propose that the user agent may
```

Comment:
implementations can

(In informative text, better use "implemetation" and talk about "user agent" in normative parts. Also "may" is RFC 2119 term and better use "can" instead in informative text.)